fe/fe-core/src/main/java/com/starrocks/connector/jdbc/PostgresSchemaResolver.java

```diff
@@ -0,0 +1,109 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.connector.jdbc;
+
+import com.google.common.collect.ImmutableSet;
+import com.starrocks.catalog.PrimitiveType;
+import com.starrocks.catalog.ScalarType;
+import com.starrocks.catalog.Type;
+import com.starrocks.common.DdlException;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.sql.Types;
+import java.util.Collection;
+
+import static java.lang.Math.max;
+
+public class PostgresSchemaResolver extends JDBCSchemaResolver {
+
+    @Override
+    public String configConnectUri(String uri, String dbName) {
```

Comment:
IMO, the biggest difference between pg and mysql is the mapping relation in starrocks.  like 
| sr | mysql | pg |
| ---- | ---- | ---- |
| catalog | instance | database |
| database | database | schema |
| table | table | table |

I think you need check that jdbc_url must contain the database when creating the pg catalog. And add some comments at the main modification place.

Reply:
thanks for your review, to ensure the consistency of users' behavior, pg connector's uri doesn't contain database， just like mysql，but i give it the default database "postgres" by configConnectorUri，is it ok?
for the mapping relation，if we want support schema for postgres，we have to make some changes of the tablename parser to support tablename like catalog.database.schema.table or some other way just like concat schema and table as tablename

Reply:
@DorianZheng PTAL

Reply:
@zombee0 I agree with @stephen-shelby, when creating catalog for PG, user should provide database of PG. And in StarRocks, schema in PG is database in SR.

Reply:
ok, i got it. thank you!